src/system.h

```diff
@@ -906,6 +906,12 @@ typedef struct {
     const Char *    cookie;
 } StructGVarFunc;
 
+#define GAP_GVAR_FUNC_TABLE_ENTRY(srcfile, name, nparam, params) \
+  {#name, nparam, \
+   params, \
+   (GVarFunc)name, \
+   srcfile ":Func" #name }
```

Comment:
In this version of the macro: absolutely!

But for some background: In other versions of the macro, the C and the GAP function have different names: Indeed, in the GAP kernel, often the C function is `FuncFOO` to implement the GAP function `FOO`.

Even then, they typically insert the name of the GAP function into the last field (so still no `Func`) there. In my packages, I decided to add the `Func`, there, though, the idea being that this string is meant to help you locate the source of a given function, so you'd really want to know the name of the C function, not of the GAP function.